src/main/java/core/basesyntax/model/dto/impl/ReportHandlerImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.model.dto.impl;
+
+import core.basesyntax.model.dto.ReportHandler;
+import core.basesyntax.storage.DataBase;
+import java.util.Map;
+
+public class ReportHandlerImpl implements ReportHandler {
+    private static final String FIRST_LINE = "fruit,quantity";
+    private static final String COLUMN_SEPARATOR = ",";
+    private Map<String, Integer> map;
+
+    public ReportHandlerImpl() {
+        map = DataBase.getDataBase();
```

Comment:
We can accept this as a constructor argument. Then it will be easier to cover the `ReportHandlerImpl` class with tests